brave/browser-laptop
# Capitalize menu items

Test Plan:
1. Open a new window in Brave and open the menu bar, if necessary.
2. Select Bravery on the menubar. Make sure two titles are exactly "Bravery Global Settings…" and "Bravery Site Settings…".
3. Select Help on the menubar. Make sure a title is exactly "Spread the Word About Brave…".
4. Select Edit on the menubar. Make sure a title is exactly "Paste Without Formatting".

Fixes #5377
